### ISSUE TITLE
[Adaptive Cards - iOS - Action.OpenURL.Json]: Voice over is conveying the irrelevant information as Group for 'This Card's action will open a URL' label.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#164]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.OpenURL.Json' and activate it.
7. Card will appear.
8. Navigate to the ''This Card's action will open a URL'' label and observe the issue.

**Actual Result:**
While navigating to the 'This Card's action will open a URL' label, screen reader is conveying the irrelevant information as group. It is announcing as **'This Card's action will open a URL Group.'**

**Expected Result:**
Screen Reader should convey the relevant label information only and it should announce the information as ''This Card's action will open a URL'' label.

**Note:**
This issue is repro throughout the application.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

### Screenshots

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.OpenURL.Json' and activate it.
7. Card will appear.
8. Navigate to the ''This Card's action will open a URL'' label and observe the issue.

**Actual Result:**
While navigating to the 'This Card's action will open a URL' label, screen reader is conveying the information twice.

**Expected Result:**
Screen Reader should convey the rellevant label information only one time and it should announce the information as ''This Card's action will open a URL'' label.

**Note:**
This issue is repro throughout the application.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/c202d357-b1cf-4a77-97ad-cc5e7026606a

### Card JSON

```JSON
NA
```
